test/library/standard/Path/prithvi-unnati/commonPrefix/testCommonPrefix.chpl

```diff
@@ -0,0 +1,27 @@
+/*
+Test cases to validate the commonPrefix([]) in Path.chpl
+*/
+
+use Path;
+
+//paths=null
+writeln(commonPrefix([""]));
```

Comment:
What happens if you send in an empty array?

Reply:
The compiler throws an error: "syntax error: near ']'"

Reply:
I think you'll need to create an empty array and then send that array to the function.  Empty arrays have domain `1..0` (so a declaration of one would look like this: `var a1: [1..0] int; writeln(commonPrefix(a1));`).

I'm also realizing we didn't add the test case to ensure the array was not modified that I mentioned a while back, which you would need a local variable array.

Reply:
I have added the testcase accordingly.